app/Http/Controllers/Three/ReviewsController.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace Rogue\Http\Controllers\Three;
+
+use Rogue\Models\Post;
+use Illuminate\Http\Request;
+use Rogue\Repositories\Three\PostRepository;
+use Rogue\Http\Controllers\Api\ApiController;
+use Illuminate\Auth\Access\AuthorizationException;
+use Rogue\Http\Transformers\Three\PostTransformer;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+
+class ReviewsController extends ApiController
+{
+    /**
+     * The post service instance.
+     *
+     * @var Rogue\Repositories\ThreePostRepository
+     */
+    protected $post;
+
+    /**
+     * @var \Rogue\Http\Transformers\Three\PostTransformer
+     */
+    protected $transformer;
+
+    /**
+     * Create a controller instance.
+     *
+     * @param  PostContract $posts
+     * @return void
+     */
+    public function __construct(PostRepository $post)
+    {
+        $this->post = $post;
+        $this->transformer = new PostTransformer;
+
+        $this->middleware('auth:api');
+    }
+
+    /**
+     * Update a post(s)'s status when reviewed.
+     *
+     * @param Request $request
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function reviews(Request $request)
+    {
+        $request->validate([
+            'post_id' => 'required',
```

Comment:
Let's check here whether a post exists: `required|exists:posts`.

Reply:
@chloealee Let's add this in before merging!

Reply:
@DFurnes shoot that got deleted by accident - re-added! :)